src/types/ErrorMessages.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessages {
+  NONE = '',
+  LOAD = 'Can\'t load todos from server',
+  TITLE = 'Title can\'t be empty',
+  ADD = 'Unable to add a todo',
+  DELETE = 'Unable to delete a todo',
+  UPDATE = 'Unable to update a todo',
```

Comment:
Usually, we do not store all the text inside the enum, enum is used for a short list of values
Display different text inside component depending on this enum